Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -344,12 +345,13 @@ def fetch_indicators_command(client, indicator_type, limit: Optional[int] = None
                     raw_json['score'] = score = client.calculate_indicator_score(risk)
                     raw_json['Criticality Label'] = calculate_recorded_future_criticality_label(risk)
                 lower_case_evidence_details_keys = []
-                evidence_details = json.loads(item.get('EvidenceDetails', '{}')).get('EvidenceDetails', [])
-                if evidence_details:
-                    raw_json['EvidenceDetails'] = evidence_details
-                    for rule in evidence_details:
-                        rule = dict((key.lower(), value) for key, value in rule.items())
-                        lower_case_evidence_details_keys.append(rule)
+                if item.get('EvidenceDetails', '{}'):
```

Comment:
`{}` is not False, are you sure this fix works? Can you add a build that failed before and passes now, with a UT for example?

Reply:
The issue is when we have for example item = {'EvidenceDetails' : None}
and then doing json.loads on None will raise an exception. 
I have added a UT for this case :) 
When I revert my fix the UT fails.

Reply:
Sounds good.
Just one comment - currently, we're getting evidence details twice.

Let's get it once, save it in a variable and use it on both line 348 and line 349